jeffvli-feishin.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Feishin
+Comment=An Electron-based music streaming app
+Exec=/home/username/.applications/Feishin-linux-x86_64.AppImage
+Icon=/home/username/.applications/Feishin-linux-x86_64.png
+Terminal=false
+Type=Application
+Categories=AudioVideo;Music;Player;
```

Comment:
Should probably also include the [`Audio` Main Category](https://specifications.freedesktop.org/menu-spec/latest/category-registry.html#:~:text=multimedia%20(audio/video)-,Audio,-An%20audio%20application) for better compatibility like here:
https://github.com/jeffvli/feishin/blob/d28054cc7f95624c983a2ce1efd1e60650ab4a45/electron-builder.yml#L69

You can keep the additional `Music` category though.